### PR TITLE
545 add support to iframes to panel component 1

### PR DESCRIPTION
**Pull request by jefferymills**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -19,22 +20,23 @@
...
     -->
-    <title>Trelliscope</title>
-  </head>
+  <title>Trelliscope</title>
+</head>
+
+<body>
+  <noscript>You need to enable JavaScript to run this app.</noscript>
+  <!-- <div id="96c61ca5" class="trelliscope-not-spa" style="width: 800px; height: 500px"></div> -->
+  <!--  <div id="1060b383" class="trelliscope-spa"></div> -->
```
Comment (Incin) [inline public/index.html:29]: did we want to commit this?

File: src/components/ContentNew/ContentNew.tsx [hunk 2]
```diff
@@ -29,6 +29,8 @@ const ContentNew: React.FC = () => {
   const basePath = useSelector(selectBasePath);
   const relatedDisplayNames = useRelatedDisplayNames();
 
+  console.log(displayInfo);
```
Comment (Incin) [inline src/components/ContentNew/ContentNew.tsx]: console log

File: src/index.tsx [hunk 3]
```diff
@@ -217,7 +217,7 @@ window.trelliscopeApp = trelliscopeApp;
 // trelliscopeApp('96c61ca5', '_test/trelliscope-examples2/gapminder_reldisp/config.jsonp', { logger: true });
 // trelliscopeApp('17a6ca23', '_test/trelliscope-examples2/network_nonraster/config.jsonp', { logger: true });
 // trelliscopeApp('96c61ca5', '/config.json', { logger: true, mockData: true });
-trelliscopeApp('960f51e6', '_test/trelliscope-examples3/gapminder_bells/config.jsonp', { logger: true });
+// trelliscopeApp('960f51e6', '_test/trelliscope-examples3/gapminder_bells/config.jsonp', { logger: true });
```
Comment (Incin) [inline src/index.tsx:220]: did you want to commit this?

**Review by Incin — CHANGES_REQUESTED** [review]

File: src/index.tsx (hunk 3 above)
Comment (jefferymills) [reply src/index.tsx:220]: Personally it feels like when we're working on a feature branch it's simpler to just not worry about these lines. I think maybe we just worry about it on develop and master. @Incin @dpenciso Thoughts?

Comment (jefferymills) [reply src/index.tsx:220]: Updated anyways so I can merge.

**Pull request merged**